defaults/main.yml

```diff
@@ -7,6 +7,8 @@ haproxy_dependencies:
     state: latest
 haproxy_install: []
 
+haproxy_restart_mode: restarted
```

Comment:
Can you name it `haproxy_restart_handler_state`?

Reply:
Yes of course, see commit b30e918